Makefile.validation

```diff
@@ -109,3 +109,24 @@ kind-load-images:
 	kind load docker-image ${PROJECT}-${EXTENDER_PATCHER}:${TAG}
 	kind load docker-image ${PROJECT}-${SCHEDULER}:${TAG}
 	kind load docker-image ${PROJECT}-${OPERATOR}:${TAG}
+
+prepare-kind:
+	# Get kind sources
+	wget -O kind-src https://codeload.github.com/kubernetes-sigs/kind/tar.gz/refs/tags/v0.11.1
+	tar -xzvf kind-src
+
+	# Add "--ipc=host" docker option and build binary
+	patch -p0 < test/kind/kind.patch
+	cd kind-0.11.1 && make build
+
+	# Copy executive file
```

Comment:
This should be a separate target. User might not have an access for this.

Reply:
created `kind-build` and `kind-copy`

Reply:
It should be `kind-install`